### DISCUSSION TITLE
Where can I read the 3.X documentation?

### DISCUSSION THREAD

**Discussion by AprildreamMI** [lerna/lerna#3489] [Q&A] [answered]
I'm trying to read the 3.X documentation but can't find it

**Comment by ghiscoding** [comment, marked as answer]
each command still have their own dedicated readme file (for example [version](https://github.com/lerna/lerna/tree/main/commands/version)), they used be available directly from the main readme page but they removed all the links from the main page and it's harder to find (I don't know why they've done that, it was helpful) and there aren't that many changes between 3.x and 6.x so you should still be able to follow the docs by just going to the [/commands](https://github.com/lerna/lerna/tree/main/commands) folder and click on the command you're interested in to see the associated docs
